11-type-constructors/latex/11-type-constructors.tex

```diff
@@ -0,0 +1,920 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
there are two 11-type-constructors.tex files, one here and one in the parent directory. which one am I reviewing? please consolidate.

Reply:
d549b0c72c3d907d22a7d90462ec39d6557f0f55

(I was using the upper .tex to generate the .md phi information)